include/vga.h

```diff
@@ -455,6 +457,7 @@ void VGA_SetupGFX(void);
 void VGA_SetupSEQ(void);
 void VGA_SetupOther(void);
 void VGA_SetupXGA(void);
+void VGA_AddCompositeSettings(Config *conf);
```

Comment:
As a general practice, should new functions like this take a reference instead of a pointer?

Reply:
Got it; thanks!